applications/FluidDynamicsApplication/custom_strategies/strategies/residualbased_predictorcorrector_velocity_bossak_scheme_turbulent.h

```diff
@@ -340,7 +310,7 @@ namespace Kratos {
 			  itNode->FastGetSolutionStepValue(DISPLACEMENT_X) = 0.0;
```

Comment:
it would make sense to acces in vector form to minimize database accesses (i understand this was pre-existing code)